src/main/resources/templates/datewarehouse/etl-jobs-config.json

```diff
@@ -213,21 +551,62 @@
 		"sourcePath" : "log.sagebase.org/userProfileSnapshots/",
 		"tableDescriptor": {
 			"name": "userprofilesnapshots",
-			"columns": {
-				"change_timestamp": "timestamp",
-				"snapshot_timestamp": "timestamp",
-				"id": "bigint",
-				"user_name": "string",
-				"first_name": "string",
-				"last_name": "string",
-				"email": "string",
-				"location": "string",
-				"company": "string",
-				"position": "string"
-			},
-			"partitionKeys": {
-				"snapshot_date": "date"
+			"description" : "This table contain snapshots of user-profiles. Snapshots are taken when user-profiles are created, updated or deleted. Note: Snapshots are also taken periodically and independently of the changes. The snapshot_timestamp records when the snapshot was taken.",
```

Comment:
Same question as above for distinguishing the difference between snapshots created via node creation/update/deletion vs the periodic snapshot taking.

Reply:
Same explanation as above